src/features/island/hud/components/inventory/Basket.tsx

```diff
@@ -41,277 +37,135 @@ export const Basket: React.FC<Prop> = ({ gameState, selected, onSelect }) => {
   const { inventory, bumpkin, collectibles } = gameState;
   const basketMap = getBasketItems(inventory);
 
-  const getCropHarvestTime = (seedName = "") => {
-    const crop = seedName.split(" ")[0] as CropName;
+  const basketIsEmpty = Object.values(basketMap).length === 0;
+  if (basketIsEmpty) {
+    return (
+      <div className="flex flex-col justify-evenly items-center p-2">
+        <img
+          src={SUNNYSIDE.icons.basket}
+          alt="Empty Chest"
+          style={{
+            width: `${PIXEL_SCALE * 12}px`,
+          }}
+        />
+        <span className="text-xs text-center mt-2">Your basket is empty!</span>
+      </div>
+    );
+  }
+
+  const isFruitSeed = (selected: InventoryItemName) =>
+    selected in FRUIT_SEEDS();
+  const isSeed = (selected: InventoryItemName) =>
+    isFruitSeed(selected) || selected in CROP_SEEDS();
+  const isFood = (selected: InventoryItemName) => selected in CONSUMABLES;
 
-    if (bumpkin) {
-      return secondsToString(
-        getCropTimeLandExpansion(
-          crop,
-          inventory,
-          collectibles,
-          bumpkin as Bumpkin
-        ),
-        {
-          length: "medium",
-        }
-      );
+  const getHarvestTime = (seedName: SeedName) => {
+    if (isFruitSeed(selected)) {
+      return SEEDS()[seedName].plantSeconds;
     }
 
-    return secondsToString(
-      getCropTime(
-        crop,
-        gameState.inventory,
-        gameState.collectibles,
-        gameState.bumpkin as Bumpkin
-      ),
-      {
-        length: "medium",
-      }
-    );
+    const crop = SEEDS()[seedName].yield as CropName;
+    return getCropTime(crop, inventory, collectibles, bumpkin as Bumpkin);
   };
 
+  const harvestCounts = getFruitHarvests(gameState);
+
   const handleItemClick = (item: InventoryItemName) => {
     onSelect(item);
   };
 
-  const basketIsEmpty = Object.values(basketMap).length === 0;
```

Comment:
<img width="510" alt="Screen Shot 2023-01-23 at 12 58 49 pm" src="https://user-images.githubusercontent.com/11745561/213954237-a2d9303e-889d-4d32-a222-9fc7e9cd5b47.png">

Have you considered what UI to show when no items are available?

I feel the previous "Your basket is empty!" was a good tip for users.

Reply:
![image](https://user-images.githubusercontent.com/107602352/213960564-3156f6f9-24a8-4c67-8d9f-92972d2c5f69.png)
I do have the screen showing when the basket/chest is empty.  On what conditions is the empty state not showing on your side?